index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/8f76becb1e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- The navbar section  start-->
+    <nav class="nav-bar">
+        <h1 class="logo-txt">My Logo</h1>
+        <i class="fa-solid fa-bars"></i>
+    </nav> 
+    <!-- header section start -->
+    <header class="header"> 
+        <div class="container">
+            <h1>
+                I'm Benjamin 
+                Glad to see   you!
+            </h1>
+            <p>
+                I'm as software developer! I can help you build a 
+                product, feature or website Look through some 
+                of my work and experience! If you like what you 
+                see and have a project you need coded, don't hesitate 
+                to contact me.
+            </p>
+            <div class="connect">
+                <h2 class="connect-txt">LET'S CONNECT</h2>
```

Comment:
- [ ] Let's kindly try to stick to the design as much as possible. The "Let's connect should be 16px". Kindly change this.

Your design:
![image](https://user-images.githubusercontent.com/86318284/176555766-d7fe5552-fd1b-4f68-bdeb-d63932e58e5e.png)

Figma design:
![image](https://user-images.githubusercontent.com/86318284/176555560-9af06dcb-1b1f-41fd-b527-9cd0cfc49d69.png)